jest.setup.js

```diff
@@ -20,4 +21,6 @@ Object.defineProperty(global, 'navigator', {
   }
 });
 
+Object.defineProperty(global, 'crypto', { value: webcrypto });
```

Comment:
crypto.getRandomValues don't come bundled with jest anymore (but it's needed for generating uuid mocks)